profile/README.md

```diff
@@ -0,0 +1,9 @@
+# Compofillers
+
+En 1972 cuatro de los mejores hombres del ejercito americano que formaban un comando, fueron encarcelados por un delito que no habían cometido.
+
+No tardaron en fugarse de la prisión en la que se encontraban recluidos.
+
+Hoy buscados todavía por el gobierno sobreviven como soldados de fortuna, si usted tiene algún problema y se los encuentra quizá pueda contratarlos…
```

Comment:
Siempre tuve la duda de si era "Y se los encuentra" o "Y si los encuentra"

Reply:
Lo copié de un blog, y he vuelto a ver el video y parece que dice "se", pero está la cosa dudosa. En inglés dice: "if you have a problem, if no one else can help, and if you can find them, maybe you can hire the A-Team" (así que el condicional sería más correcto xD)